dashboard/config/libraries/zGameDev.interpreted.js

```diff
@@ -1,32 +1,16 @@
 var GRID_SIZE = 8;
 var CELL_SIZE = 400 / GRID_SIZE;
-var GRAVITY = -0.5;
+var GRAVITY = -0.25;
 
 setDefaultSpriteSize(CELL_SIZE);
 
 // Handles sprite interactions every frame.
 function gameDevLoop() {
   collide('collide', {group: 'players'}, {group: 'walls'});
-  collide('collide', {group: 'objects'}, {group: 'walls'});
-  collide('collide', {group: 'enemies'}, {group: 'walls'});
-  collide("collide", ({costume: "all"}), ({group: "walls"}));
- 
+  collide('collide', {group: ''}, {group: 'walls'});
```

Comment:
for my understanding: a group of `''` means the sprite has no group? How does making this change mean wall sprites don't collide with wall sprites (what was making that happen before)?

Reply:
The `group` property is undefined for sprites unless explicitly set. We're using `''` as a way to signal that we want to return the ones without a group. What was doing it before was this removed line:

`collide("collide", {costume: "all"}, {group: "walls"});`

Which can be read "all sprites collide with walls". Now, it's just "all sprites without a group collide with walls".

Reply:
It's currently only possible to set a value to `sprite.group` using a dropdown, and none of the options use an empty string.

Reply:
thanks! It might be worth adding a comment explaining this.

Reply:
Will add!